e2e/completeFlow.e2e.ts

```diff
@@ -0,0 +1,158 @@
+import axios from 'axios';
+import {by, device, expect, element} from 'detox';
+
+// In a real-world scenario, this would be stored in a secure location, like an environment variable
+const testCredentials = {
+  username: 'test@groundswell.io',
+  password: 'Temp1234!',
```

Comment:
Not in an environment variable as the credentials were already exposed in the readme, but ideally should - it only goes a bit out of scope.